icgcget/clients/download_client.py

```diff
@@ -43,6 +69,9 @@ def _run_command(self, args, parser, env=None):
         if None in args:
             self.logger.warning("Missing argument in %s", args)
             return 1
+        if not env:
+            env = dict(os.environ)
+        env['PATH'] = '/usr/local/bin:' + env['PATH']  # virtalenv compatibility
```

Comment:
Why is that? Is it because virtualenv modifies the path? What breaks if this line is missing?